edgelet/iotedge/src/config/import/old_config/mod.rs

```diff
@@ -109,15 +109,57 @@ pub(crate) struct Certificates {
 
 #[derive(Debug, serde_derive::Deserialize)]
 pub(crate) struct DeviceCertificate {
-    pub(crate) device_ca_cert: String,
-    pub(crate) device_ca_pk: String,
-    pub(crate) trusted_ca_certs: String,
+    #[serde(deserialize_with = "deserialize_file_uri_or_path")]
+    pub(crate) device_ca_cert: Url,
+    #[serde(deserialize_with = "deserialize_file_uri_or_path")]
+    pub(crate) device_ca_pk: Url,
+    #[serde(deserialize_with = "deserialize_file_uri_or_path")]
+    pub(crate) trusted_ca_certs: Url,
 }
 
 const fn default_auto_generated_ca_lifetime_days() -> u16 {
```

Comment:
nit: any reason this is a constant fn vs. a regular constant? If not, might as well switch it over (since we don't have that many const functions in our codebase anyways)

Reply:
`serde(deserialize_with)` needs a path, not an expr.

Reply:
Ah, the git diff cut off the relevant `serde(deserialize_with)`. All good then.